__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const db  = require("../db/connection");
+const app = require("../db/app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data")
+const request = require("supertest")
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("get/api/topics", () => {
+  test("Return topics", () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toHaveLength(3);
```

Comment:
This is really nice. It's not enough to have just the forEach below, because if we get 0 topics back, it won't fail because it won't find any cases where there is no description or slug. By also checking the length, you're ensuring we don't get any false positives and it makes for much more reliable code.

Reply:
Now would also be a good time to include a test (outside of your api/topics block) to check that you get a 404 if someone tries to go to an endpoint that doesn't exist such as `/api/hello`

Reply:
Im sorry I dont know what you mean like in what test I should add here, wont it fail with 0 results as its expecting length 3, what are you suggesting I add? I dont really have any tests Ive done previously with much more than this on for this kind of thing.